### PR TITLE
PAYARA-477

### PR DESCRIPTION
**Pull request by MeroRai**
INFO level log moved to FINE level.

### CODE REVIEW

**Comment by danPowell52** [comment]
payara test please

**Comment by danPowell52** [comment]
payara test please

**Comment by danPowell52** [comment]
payara test please

**Comment by danPowell52** [comment]
payara test please

**Comment by danPowell52** [comment]
payara test please

**Comment by danPowell52** [comment]
payara test please

**Comment by Pandrex247** [comment]
payara test please

**Comment by danPowell52** [comment]
Jenkins Test

**Comment by danPowell52** [comment]
Dan Test

**Comment by danPowell52** [comment]
Jenkins Test

**Comment by danPowell52** [comment]
Jenkins Test

**Comment by danPowell52** [comment]
Jenkins Test

**Comment by danPowell52** [comment]
payara test please

**Comment by danPowell52** [comment]
Jenkins test please

**Comment by danPowell52** [comment]
payara test please

**Comment by danPowell52** [comment]
Jenkins Test

**Comment by danPowell52** [comment]
payara test please

**Comment by danPowell52** [comment]
Jenkins Test

**Comment by danPowell52** [comment]
Jenkins Test

**Comment by danPowell52** [comment]
build please

**Comment by danPowell52** [comment]
build please

**Comment by danPowell52** [comment]
Pull Request Test

**Comment by danPowell52** [comment]
Pull Test Please

**Comment by danPowell52** [comment]
Pull Test Please

**Comment by danPowell52** [comment]
Pull Test Please

**Comment by danPowell52** [comment]
Pull Test Please

**Comment by danPowell52** [comment]
Pull Test Please

**Comment by danPowell52** [comment]
Flow Test Please

**Comment by danPowell52** [comment]
Flow Test Please

File: appserver/web/weld-integration/src/main/java/org/glassfish/cdi/transaction/TransactionalInterceptorBase.java [hunk 1]
```diff
@@ -96,72 +98,72 @@
...
     @LogMessageInfo( message = "Managed bean with Transactional annotation and TxType of REQUIRED " +
                         "called outside a transaction context.  Beginning a transaction...",
-                     level = "INFO")
+                     level = "FINE")
     public static final String CDI_JTA_MBREQUIRED = "AS-JTA-00010";
 
     @LogMessageInfo( message = "Managed bean with Transactional annotation and TxType of REQUIRED " +
                                "encountered exception during begin {0}",
-                     level = "INFO")
+                     level = "FINE")
```
Comment (Pandrex247) [inline appserver/web/weld-integration/src/main/java/org/glassfish/cdi/transaction/TransactionalInterceptorBase.java]: Do we want the messages like this (lines 131, 136, 141, 145, 150, 155, and 160) to be at FINE level? They look like error messages to me. Leaving at INFO or changing to WARNING might be more appropriate.

Comment (smillidge) [inline appserver/web/weld-integration/src/main/java/org/glassfish/cdi/transaction/TransactionalInterceptorBase.java]: I agree these need to be changed

**Comment by danPowell52** [comment]
jenkins test please

**Comment by danPowell52** [comment]
Flow Test Please

**Comment by danPowell52** [comment]
jenkins test please

**Comment by danPowell52** [comment]
jenkins test please

**Comment by MeroRai** [comment]
jenkins test please

**Comment by danPowell52** [comment]
pull this

**Comment by danPowell52** [comment]
pull this please

**Comment by danPowell52** [comment]
pull test this please

**Comment by danPowell52** [comment]
WE WILL TEST

**Comment by danPowell52** [comment]
jenkins test please

**Comment by danPowell52** [comment]
jenkins test please

**Comment by danPowell52** [comment]
jenkins test please

**Comment by Pandrex247** [comment]
fixes #481

**Comment by Pandrex247** [comment]
Flow Test Please

**Pull request merged**